ecs-cli/modules/commands/compose/service/compose_service_command.go

```diff
@@ -182,6 +182,24 @@ func serviceDiscoveryFlags() []cli.Flag {
 	}
 }
 
+func updateServiceDiscoveryFlags() []cli.Flag {
+	return []cli.Flag{
+		cli.BoolFlag{
+			Name:  flags.UpdateServiceDiscoveryFlag,
+			Usage: "[Optional] Service Discovery - Update Service Discovery related settings. Only the Service Discovery Service settings DNS TTL and Failure Threshold can be updated.",
```

Comment:
Oh good catch, that's because originally the same flag was going to be used for create and update, but then I decided against that to prevent confusion.